index.js

```diff
@@ -0,0 +1,23 @@
+const enableModal = document.querySelector('#menu');
+const body = document.querySelector('body');
+const extBtn = document.getElementById('exit-btn');
+const modal = document.querySelector('#modal');
+const modalLinks = document.querySelector('.modal-nav-link');
+
+enableModal.addEventListener('click', () => {
+  modal.classList.remove('display-none');
+  body.classList.add('none-scroll');
+});
+
+extBtn.addEventListener('click', () => {
+  modal.classList.add('display-none');
+  body.classList.remove('none-scroll');
+});
+
+for (let i = 0; i < modalLinks.length; i += 1) {
+  modalLinks[i].addEventListener('click', () => {
+    modal.classList.add('display-none');
+    body.classList.remove('none-scroll');
+    body.classList.add('scroll');
```

Comment:
- [ ] Great job so far 👏🏻 when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. However, I have noticed that this prevents your page to scroll since you're adding an `overflow: hidden;` but you don't forget to remove this after clicking the mobile menu options. Kindly fix this here is your CSS 👇🏻 

```css
.none-scroll {
  overflow: hidden;
}

.none-scroll {
  overflow: hidden;
}
```